src/ast/CMakeLists.txt

```diff
@@ -5,8 +5,7 @@ add_library(verona-ast-lib
   err.cc
   files.cc
   parser.cc
-  sym.cc
-  )
+  sym.cc)
```

Comment:
small nit: avoid unnecessary changes on files that have no functional change. We always want to touch the least number of files on every commit.

Reply:
Thanks, this was left over from the initial commit where more changes were made to that file.